docs/SOPs/dataset_suitability_SOP.md

```diff
@@ -7,7 +7,13 @@ parent: SOPs
 # HCA Dataset Eligibility criteria 
 
 ## Purpose of this document
-As wranglers, our current method of tracking new, in-progress (wrangling) and complete datasets is by indicating their wrangling status, priority and release in Ingest. When new datasets are added, we need to assess whether they are eligible for HCA. There are specific eligibility criteria that we use to decide whether a dataset is eligible or not. It is important that this information is available so that time is not wasted on wrangling datasets that are not eligible for HCA and/or cannot be ingested in our system. The purpose of this document is to list the specific criteria we use to assess dataset eligibility and our general approach to implement these. Any feedback on this process is very welcome and should be added to this document; please make it clear that it is feedback.
+As wranglers we track new, in-progress (wrangling) and complete datasets by adding them as projects in ingest. You can [add a project](https://contribute.data.humancellatlas.org/projects/register/autofill) automatically with a doi or an accession, or you can register one manually.
+
+When we register a new project we need to indicate its priority, eligibility, status and other key information in ingest (see the dataset [acknowledgment SOP](https://ebi-ait.github.io/hca-ebi-wrangler-central/SOPs/dataset_acknowledgement_SOP.html#manual-curation-after-addition)). Having all this information helps us to select quickly projects to wrangle and keeps us from wasting time on projects that cannot be ingested and/or are not eligible for the HCA.
+
+The purpose of this document is to list the specific criteria we use to assess dataset eligibility and our general approach to implement these. Any feedback on this process is very welcome and should be added to this document; please make it clear that it is feedback. In the longer term we are aiming for a software focused solution to this task.
```

Comment:
We should include how to add feedback (e.g. mail the wrangler team at wrangler-team@data.humancellatlas.org)

Reply:
We should add the email to a user facing document, maybe the submission guidelines?